src/js/components/Form/stories/RequiredLabel.js

```diff
@@ -2,34 +2,23 @@ import React from 'react';
 
 import { Box, Button, Form, FormField, Grommet, Text } from 'grommet';
 import { grommet } from 'grommet/themes';
+import { deepMerge } from 'grommet/utils';
 
-const FormFieldLabel = props => {
-  const { required, label, ...rest } = props;
-  return (
-    <FormField
-      label={
-        required ? (
-          <Box direction="row">
-            <Text>{label}</Text>
-            <Text color="status-critical">*</Text>
-          </Box>
-        ) : (
-          label
-        )
-      }
-      required={required}
-      {...rest}
-    />
-  );
-};
+const customTheme = deepMerge(grommet, {
+  formField: {
+    label: {
+      requiredAsterisk: true,
+    },
+  },
+});
 
 export const RequiredLabel = () => (
-  <Grommet theme={grommet}>
+  <Grommet theme={customTheme}>
     <Box align="center" pad="large">
       <Form>
-        <FormFieldLabel name="firstName" label="FirstName" required />
-        <FormFieldLabel name="LastName" label="LastName" required />
-        <FormFieldLabel name="email" label="Email" />
+        <FormField name="firstName" label="First Name" required />
```

Comment:
Should these be presented as: 
```
<FormField htmlFor="firstName" ... />
  <TextInput id="firstName" ... />
</FormField>
```
?

I thought we had been trying to promote this pattern as opposed to just `<FormField />`. If just FormField,
```suggestion
        <FormField htmlFor="firstName" id="firstName" name="firstName" label="First Name" required />
```

Either way, I think we want id and htmlFor so the field has an associated label and passes WAVE test.

Reply:
Yes, I can enhance to our recommended format.

Reply:
Done.